includes/api/class-edd-api.php

```diff
@@ -92,7 +92,43 @@ class EDD_API {
 	 * @access private
 	 * @since 1.7
 	 */
-	private $override = true;
+	public $override = true;
+
+	/**
+	 * Version of the API queried
+	 *
+	 * @var string
+	 * @access public
+	 * @since 2.4
+	 */
+	private $queried_version;
```

Comment:
Same for this. Particularly useful if you wanted to say write an extension that logs api requests outside of the edd core one.